tests/integration/models/reddit/test_comment.py

```diff
@@ -76,10 +75,10 @@ def test_enable_inbox_replies(self):
     def test_gild__no_creddits(self):
         self.reddit.read_only = False
         with self.recorder.use_cassette("TestComment.test_gild__no_creddits"):
-            with pytest.raises(BadRequest) as excinfo:
+            with pytest.raises(RedditAPIException) as excinfo:
                 Comment(self.reddit, "d1616q2").gild()
-            reason = excinfo.value.response.json()["reason"]
-            assert "INSUFFICIENT_CREDDITS" == reason
+            exception = excinfo.value
+            assert "INSUFFICIENT_CREDDITS" == exception.error_type
```

Comment:
Assuming that Creddits are no longer used, wouldn't this exception actually return something else now? It should be re-recorded.

Reply:
You make a good point about this error_type. However, I'm going to skip on regenerating it as the concern of this PR is to ensure these exceptions are captured by `RedditAPIExceptions` regardless of their types.

Reply:
I'll bump this to a seperate issue.